src/cascade/dismod/db/wrapper.py

```diff
@@ -161,8 +161,10 @@ def __init__(self, engine, avgint_columns, data_columns):
         self._table_definitions = self._metadata.tables
         self._table_data = {}
         self._table_hash = {}
-        add_columns_to_avgint_table(self._metadata, avgint_columns)
-        add_columns_to_data_table(self._metadata, data_columns)
+        if avgint_columns:
```

Comment:
I'm avoiding having mutable default arguments on principle, though it actually wouldn't be a problem here unless someone changed the implementation. Though it would be a problem if they changed the implementation... Also the linter will yell at me if I do it. Hence the conditionals.